src/js/include/llm_pipeline/llm_pipeline_wrapper.hpp

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <future>
```

Comment:
Adding a header without changing any code in the file is strange. Remove it

Reply:
Sorry, I thought this include should be set in the header file, not in the source file. 
I have fixed that.